src/overlays/actors/ovl_En_Recepgirl/z_en_recepgirl.c

```diff
@@ -144,7 +144,7 @@ void EnRecepgirl_Talk(EnRecepgirl* this, GlobalContext* globalCtx) {
     if (temp_v0_2 == 2) {
```

Comment:
You can name this temp now too (and some of this probably should change in the tutorial since this file is the one that's documented).

Reply:
For this:
`temp_v0_2 = Message_GetState(&globalCtx->msgCtx);`
I see both:
`dialogState = Message_GetState(&globalCtx->msgCtx);`
and
`talkState = Message_GetState(&globalCtx->msgCtx);`
in both MM and OoT. We should probably consolidate those at some point.

Reply:
Ah, I hadn't noticed it was inconsistent. Leave it for now, then.